src/main/kotlin/de/mariushoefler/flutterenhancementsuite/actions/CustomFlutterTemplateManager.kt

```diff
@@ -0,0 +1,55 @@
+package de.mariushoefler.flutterenhancementsuite.actions
+
+import com.intellij.ide.actions.CreateFileFromTemplateDialog
+import com.intellij.ide.fileTemplates.FileTemplate
+import com.intellij.ide.fileTemplates.impl.CustomFileTemplate
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.project.guessProjectDir
+import icons.FlutterIcons
+import java.io.File
+
+private const val CUSTOM_TEMPLATES_PATH = ".idea/customFlutterFileTemplates/"
```

Comment:
I would lift this out of `.idea` as many teams ignore this directory in their .gitignore and also simplify the name e.g. "flutter_file_templates"

Reply:
I went for a name with a dot, is that fine?